docs/source-pytorch/common/trainer.rst

```diff
@@ -33,10 +33,22 @@ This is the basic use of the trainer:
 
 .. code-block:: python
 
-    model = MyLightningModule()
+    import lightning.pytorch as pl
+    from lightning.pytorch import Trainer
```

Comment:
I suggest to reduce this import
```suggestion
```
and below use

pl.Trainer()

Reply:
I had done this previously and reverted because there are some many instances of `trainer = Trainer()` on the page that it makes sense to leave this as is until I can update the docstrings that also have `trainer = Trainer()` .

Reply:
implemented in [7cf7aff](https://github.com/Lightning-AI/lightning/pull/18452/commits/7cf7aff92dddb21c33efac1902289d313658a7f4)